wait/sql.go

```diff
@@ -12,7 +12,7 @@ import (
 const defaultForSqlQuery = "SELECT 1"
 
 //ForSQL constructs a new waitForSql strategy for the given driver
-func ForSQL(port nat.Port, driver string, url func(string, nat.Port) string) *waitForSql {
+func ForSQL(port nat.Port, driver string, url func(host string, port nat.Port) string) *waitForSql {
```

Comment:
I'm adding this cosmetic change here, but I can send it in a separate PR if needed

Reply:
this is a breaking change?